mlflow/protos/databricks_uc_registry_messages.proto

```diff
@@ -26,6 +26,17 @@ message RegisteredModel {
 
   // Description of this ``registered_model``.
   optional string description = 5;
+
+  // Registered Model Aliases on this model.
+  repeated RegisteredModelAlias aliases = 6;
+}
+
+message RegisteredModelAlias {
+  // Unique name for the alias.
```

Comment:
```suggestion
  // The name of the alias
```

Or if we want to mention uniqueness maybe clarify that the alias name is unique within a registered model?